Author: Artjom Slepnjov "Shellgen"
Description: Ignore a cannot create netlink socket: Permission denied
Ver: dnsmasq-2.91 or newer version.
Date: 2026-09-30 08:00 UTC
==============================================================
--- a/dev/null
+++ b/src/stub-netlink.c
@@ -0,0 +1,6 @@
+/* stub-netlink.c */
+#include "dnsmasq.h"
+void netlink_init(void) { return; }  /* no-op */
+int netlink_process(void) { return 0; }  /* nothing to do */
+void netlink_multicast(void) { return; }  /* no-op */
+void iface_enumerate(void) { return; }  /* no-op */
